src/transformers/models/swin/configuration_swin.py

```diff
@@ -75,6 +75,8 @@ class SwinConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        add_final_layer_norm (`bool`, *optional*, defaults to True):
+            Whether or not to add a final layer normalization layer after the stages.
```

Comment:
No we don't add those kind of parameters to configs, this means a new architecture should be created.

Reply:
This is resolved as there's now a separate `DonutSwinModel`.